### PR TITLE
disentangle jsons for flp and epn to avoid conflict in TOF matching QC

### PR DESCRIPTION
**Pull request by noferini**
This is to disentangle the jsons for flp and epn loaded from consul otherwise we didn't find any way to avoid conflicts (epn needs the possibility to adjust TOF matching QC setting independently of FLPs: with TRD w/o TRD)

At the moment we just cloned flp json (already done) so the situation is not yet changed wrt before (we just disentagled the two jsons).
Once this PR is available in EPNs we can run a test with synthetic runs.

@njacazio @ercolessi

### CODE REVIEW

**Comment by chiarazampolli** [comment]
Ciao @noferini , 
So on FLPs, you will load the old names?
`consul://o2/components/qc/ANY/any/tof-full-qcmn`
and 
`consul://o2/components/qc/ANY/any/tof-full-epn-qcmn`
?
Just to be sure, since from this PR is not clear how the issue is fixed.
Thanks!
Chiara

**Comment by noferini** [comment]
> Ciao @noferini , So on FLPs, you will load the old names? `consul://o2/components/qc/ANY/any/tof-full-qcmn` and `consul://o2/components/qc/ANY/any/tof-full-epn-qcmn` ? Just to be sure, since from this PR is not clear how the issue is fixed. Thanks! Chiara

This PR doesn't touch what is done on FLP. Indeed we will continue to load old jsons

**Comment by chiarazampolli** [comment]
Exactly, this is what I meant. 
Thanks! I am approving now.

**Review by chiarazampolli — APPROVED** [review]

**Comment by chiarazampolli** [comment]
Is FST accessing the files in consul? Should we wait for the test before merging?

**Comment by noferini** [comment]
> Is FST accessing the files in consul? Should we wait for the test before merging?

It shouldn't (and I guess it is not possible outside CERN).
We can merge it

**Comment by chiarazampolli** [comment]
Done.

**Pull request merged**
